src/test/java/org/terasology/blockdetector/systems/BlockDetectorSystemTest.java

```diff
@@ -86,6 +88,7 @@ public void detectorTest(){
         Region3i nonAerialRange = Region3i.createFromMinMax(new Vector3i(-3, -3, -3), new Vector3i(3, 3, 3));
         data.setNonAerialRange(nonAerialRange);
 
+        //Run the addDetector method
```

Comment:
You are adding redundant comments. Anyone can see the next line which is calling the addDetector method. Why did you feel the need to add this comment here?